packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to Okta which handles the flow in its own window, or handle each step in the flow by building the appropriate user interface elements, capturing the information, and updating Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authentication methods (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authentication methods are required (multifactor), and which methods are optional. This results in a large number of possible combinations for the different factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta Application Integration, request the initial step in the flow, and cycle through responding to steps until either the user is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+
+- Choosing an authenticator, such as Okta Verify or security questions.
+- Entering a One-Time Passcode (OTP).
+- Cancelling the sign-in flow.
+
+## Sign-in objects
+
+Each step in the sign-in flow is represented by a number of different SDK objects:
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+- **Response:** The top-level object that represents a step and contains all the other objects. It also indicates a successful sign-in, and is used to send messages for cancelling the sign-in flow, or to retrieve an access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
+- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. It's also used to update values entered by the user and to request the next step in the flow.
+- **Authenticator:** Represents a type, or factor for verifying the identity of a user, such as a username and password.
```

Comment:
Username isn't really a factor.  The authenticator, in this case, would be like "Phone", or "Email"

Reply:
I'm a bit confused by that. I thought authenticators where things like Okta Verify, security questions, and such.... am I missing something?

Reply:
Correct. You say "Such as a username and password", which is misleading.  A username isn't an authenticator, but a part of identity.  The password part _is_ an authenticator.

Reply:
This will be in the next commit:

> - **Authenticator:** Represents a type, or factor for verifying the identity of a user, such as a password or OTP.